ports/stm32/boards/ARDUINO_PORTENTA_H7/board_init.c

```diff
@@ -162,7 +162,7 @@ void PORTENTA_board_low_power(int mode) {
             break;
     }
 
-    #if MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE
+    #if MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE == 0
```

Comment:
This should be `#if !MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE`.

Reply:
Changed, however I think `==` is more readable than `!` which is very easy to miss, also it's common to check for `== 0` or `== 1` in the code base, for example just in the `stm32` port: `__DCACHE_PRESENT` , `__CORTEX_M`, `USBH_USE_OS` etc..

Reply:
Thanks for changing.

`__DCACHE_PRESENT` is the way it is because that's following how it's used in CMSIS.

`__CORTEX_M` needs to use == because it can be different values, eg 3, 4, 7.

`USBH_USE_OS` is originally code from STM32.

If there are any cases of using == for MicroPython macros which are simply enable/disable then we should change those because it's not consistent.

Reply:
Okay will avoid it in the future, but yes there are some instances, for example:

Unix port: `unix/main.c:#if MICROPY_USE_READLINE == 1`
nrf port: `nrf/mphalport.c:#if defined(NRFX_LOG_ENABLED) && (NRFX_LOG_ENABLED == 1)`
samd: `samd/samd_isr.c:    #if __FPU_PRESENT == 1 && __FPU_USED == 1`
mimxrt port: `mimxrt/boards/MIMXRT1176_clock_config.c:    #if defined(XIP_BOOT_HEADER_ENABLE) && (XIP_BOOT_HEADER_ENABLE == 1)`